Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
The game does not need MMU mode.  It'd be a shame to force it because of an errant DCBZ not handled by just one of the CPU cores.  I'd be more inclined to remove the optimisation in this case.

Out of the options so far, I'm leaning more towards masking on RAM_MASK before the PXOR if !bWii